temporian/core/operators/simple_moving_average.py

```diff
@@ -66,6 +62,13 @@ def __init__(
   def build_op_definition(cls) -> pb.OperatorDef:
     return pb.OperatorDef(
         key="SIMPLE_MOVING_AVERAGE",
+        attributes=[
+            pb.OperatorDef.Attribute(
+                key="window_length",
+                type=pb.OperatorDef.Attribute.Type.STRING,
+                is_optional=False,
```

Comment:
s/False/True ?

Reply:
If this means substituting False for True: isn't window length mandatory? Do we want it to have a default value?

Reply:
My bad, I was looking at sampling. This looks good.